Jellyfin.Plugin.Anime/Providers/GenreHelper.cs

```diff
@@ -50,21 +50,6 @@ public static class GenreHelper
             {"Yaoi", "Adult"},
             {"Yuri", "Adult"},
```

Comment:
Not really related to the changes in the PR, but I'm not a fan of this mapping.

To me, Yaoi or Yuri doesn't necessarily imply an "Adult" genre (i.e. it's not automatically porn). But it's also not a genre (Same with Shounen, Seinen and Josei, they're not genres but target audiences).

The "correct" way, imo, for this along with "genres" like Vampire, Witch, Shounen, Josei, Mecha, etc is to have the few general genres (Action, Adventure, Animation, Supernatural, etc) then the specifics as Tags.

So something like Mobile Suit Gundam would have genres as "Action, Science-Fiction, Animation" and tags like "Mecha, Space Opera".

Reply:
I'd say this is out of scope for these changes.

Reply:
For sure, as I said, it's more of a thought I had when reading the source :p

It seems to already work that way, but some of the mappings don't make sense. I'll whip something up when I have some time.